test-network/network.sh

```diff
@@ -20,40 +20,45 @@ export VERBOSE=false
 function printHelp() {
   echo "Usage: "
   echo "  network.sh <Mode> [Flags]"
-  echo "    <Mode>"
-  echo "      - 'up' - bring up fabric orderer and peer nodes. No channel is created"
-  echo "      - 'up createChannel' - bring up fabric network with one channel"
-  echo "      - 'createChannel' - create and join a channel after the network is created"
-  echo "      - 'deployCC' - deploy the fabcar chaincode on the channel"
-  echo "      - 'down' - clear the network with docker-compose down"
-  echo "      - 'restart' - restart the network"
+  echo "    Modes:"
+  echo "      "$'\e[0;32m'up$'\e[0m' - bring up fabric orderer and peer nodes. No channel is created
+  echo "      "$'\e[0;32m'up createChannel$'\e[0m' - bring up fabric network with one channel
+  echo "      "$'\e[0;32m'createChannel$'\e[0m' - create and join a channel after the network is created
+  echo "      "$'\e[0;32m'deployCC$'\e[0m' - deploy the asset transfer basic chaincode on the channel or specify
+  echo "      "$'\e[0;32m'down$'\e[0m' - clear the network with docker-compose down
+  echo "      "$'\e[0;32m'restart$'\e[0m' - restart the network
   echo
   echo "    Flags:"
   echo "    -ca <use CAs> -  create Certificate Authorities to generate the crypto material"
   echo "    -c <channel name> - channel name to use (defaults to \"mychannel\")"
   echo "    -s <dbtype> - the database backend to use: goleveldb (default) or couchdb"
   echo "    -r <max retry> - CLI times out after certain number of attempts (defaults to 5)"
   echo "    -d <delay> - delay duration in seconds (defaults to 3)"
-  echo "    -l <language> - the programming language of the chaincode to deploy: go (default), java, javascript, typescript"
-  echo "    -v <version>  - chaincode version. Must be a round number, 1, 2, 3, etc"
+  echo "    -ccn <name> - the short name of the chaincode to deploy: basic (default), secured, ledger, private"
+  echo "    -ccl <language> - the programming language of the chaincode to deploy: go (default), java, javascript, typescript"
+  echo "    -ccn <name> - the short name of the chaincode to deploy: basic (default), secured, private"
```

Comment:
-ccn is added twice here.

Also, let's keep the sample names in the same logical order as the directory structure - basic (default), ledger, private, secured

Reply:
done